java/feathub-test-proto/src/proto/message.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package org.feathub.proto;
+option java_package = "org.feathub.proto";
+option java_multiple_files = true;
+
+message FileSystemTestMessage {
+  // TODO: We have to add a dummy field with type int64 at the beginning,
+  //  otherwise exception will be thrown.
+  //  https://issues.apache.org/jira/browse/FLINK-31944
+  int64 dummy = 1;
+  string name = 2;
+  int64 cost = 3;
+  int64 distance = 4;
```

Comment:
Can we have a test that covers all supported types? Maybe just add fields in this message and verify that the fields can be written and read correctly?